test/unit/ajax.js

```diff
@@ -1578,16 +1578,19 @@ module( "ajax", {
 		}
 	} );
 
-	ajaxTest( "#13240 - jQuery.ajax() - support non-RFC2616 methods", 1, {
-		url: "data/echoQuery.php",
-		method: "PATCH",
-		success: function() {
-			ok( true, "success" );
-		},
-		error: function() {
-			ok( false, "error" );
-		}
-	});
+	// BrowserStack PATCH support sometimes breaks so on TestSwarm run the test in IE8 only.
```

Comment:
Well, I've already exchanged 22 mails with BrowserStack people, the first one at the end of October. This problem disappears and then re-appears again after some time. I'd rather not spend more time on this so it's better to restrict this to IE8.

As for switching from TestSwarm: sure, though if we forget about it then nothing terrible will happen. We'll notice it the first time PATCH fails again. ;) And then we'll fix the exclusion logic.